homework_14_140123/src/MethodsImplementation.java

```diff
@@ -0,0 +1,409 @@
+                    i = i + 11;
+                    arrString[i] = arrString[i] + "\n";
+                } else {
+                    arrString[i] = arrString[i] + "\n";
+                }
+            }
+            String line2 = String.join("", arrString);
+            bufferedWriter.write(line2);
+        } catch (IOException e) {
+            System.out.println(e);
+        } finally {
+            try {
+                bufferedWriter.close();
+            } catch (IOException e) {
+                System.out.println(e);
+            }
+        }
+        BufferedReader bufferedReader = null;
+        try {
+            bufferedReader = new BufferedReader(new FileReader(file));
+            StringBuilder stringBuilder = new StringBuilder();
+            while (true) {
+                int i;
+                if ((i = bufferedReader.read()) == -1) {
+                    break;
+                } else {
+                    stringBuilder.append((char) i);
+                }
+            }
+            System.out.println(stringBuilder);
+        } catch (IOException e) {
+            System.out.println(e);
+        } finally {
+            try {
+                bufferedReader.close();
+            } catch (IOException e) {
+                System.out.println(e);
+            }
+        }
+    }
+
+    //Task7
+//    Создать текстовый файл с именем original.txt,
+//    который будет содержать строку "HellO from text filE",
+//    далее необходимо перезаписать содержимое этого файла в новый файл с именем originalCOPY.txt,
+//    удалив все гласные из строки.
+    public static void copyFile() {
+        File file = new File("original.txt");
+        BufferedWriter bufferedWriter = null;
+        String line = "";
+        try {
+            file.createNewFile();
+            bufferedWriter = new BufferedWriter(new FileWriter(file, true));
+            bufferedWriter.write("HellO from text filE");
+        } catch (IOException e) {
+            System.out.println(e);
+        } finally {
+            try {
+                bufferedWriter.close();
+            } catch (IOException e) {
+                System.out.println(e);
+            }
+        }
+        BufferedReader bufferedReader = null;
+        File fileCopy = new File("originalCOPY.txt");
+        BufferedWriter bufferedWriter2 = null;
+        try {
+            bufferedReader = new BufferedReader(new FileReader(file));
+            int i;
+            do {
+                i = bufferedReader.read();
+                if (i != -1) {
+                    line += (char) i;
+                }
+            } while (i != -1);
+            System.out.println(line);
+            bufferedWriter2 = new BufferedWriter(new FileWriter(fileCopy));
+            String newline = line.replaceAll("[aeiouAEIOU]", "");
+            bufferedWriter2.write(newline);
+        } catch (IOException e) {
+            System.out.println(e);
+        } finally {
+            try {
```

Comment:
you don't need try block here, just close stream

Reply:
fixed. using try with resources.